airflow/utils/tests.py

```diff
@@ -71,15 +75,70 @@ class Dummy3TestOperator(BaseOperator):
     operator_extra_links = ()
 
 
+@attr.s(auto_attribs=True)
+class CustomBaseIndexOpLink(BaseOperatorLink):
```

Comment:
```suggestion
class CustomIndexOpLink(BaseOperatorLink):
```

please -- it's not a Base class :)